docs/development.md

```diff
@@ -0,0 +1,23 @@
+# GW Launcher Development
+## GitHub Actions Build Pipeline
+GW Launcher build pipeline can be found [here](../.github/workflows/build.yml).
+
+Steps:
+- Calculate version using [GitVersion](https://gitversion.net/) based on the Git tag
+- Build GW Launcher using the `FrameworkDependent` [publish profile](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/visual-studio-publish-profiles)
```

Comment:
should probably rename this

Reply:
Happy to make that change as part of this PR - do you have any preferences around the naming?

Reply:
anything that makes sense, no specific preference

Reply:
That sounds descriptive enough. Updated accordingly!